public/components/custom_panels/panel_modules/panel_grid.tsx

```diff
@@ -50,9 +53,12 @@ export const PanelGrid = ({
   startTime,
   endTime,
   onRefresh,
+  cloneVisualization,
+  deleteVisualization,
+  pplFilterValue,
 }: Props) => {
-  const [layout, setLayout] = useState([]);
-  const [editedLayout, setEditedLayout] = useState([]);
+  const [layout, setLayout] = useState([] as Layout[]);
```

Comment:
not an issue but i think the syntax is usually `useState<Layout[]>([])`?

Reply:
will change this